src/main/java/com/univ/lille/copainsderoute/platine/service/EventService.java

```diff
@@ -198,7 +198,12 @@ public List<Event> getEventsByLocation(GpsCoordinatesDTOs gpsCoordinatesDTOs) {
                 eventsByLocation.add(event);
             }
         }
-        return eventsByLocation;
+        List<EventResponseDTOs> eventResp = new ArrayList<>();
+        for(Event event : eventsByLocation) {
+        	eventResp.add(new EventResponseDTOs(event));
+        }
```

Comment:
Tu dois pouvoir faire ça directement si tu veux
```suggestion
        List<EventResponseDTOs> eventResp = eventsByLocation.stream().map(EventResponseDTOs::new).toList();
```